From: Andrey Borzenkov <arvidjaar@gmail.com>
Date: Fri, 21 Feb 2014 20:44:06 +0400
Subject: man: fix description of systemctl --after/--before

It was backward - --after fetches After property, so units shown really
come *before* unit given as argument. Same for --before.

(cherry picked from commit 4a77ca700d1841c2b8a86ed5b7f3495db61c236d)
---
 man/systemctl.xml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/man/systemctl.xml b/man/systemctl.xml
index 355cd11b..fef95789 100644
--- a/man/systemctl.xml
+++ b/man/systemctl.xml
@@ -148,8 +148,8 @@ along with systemd; If not, see <http://www.gnu.org/licenses/>.
         <term><option>--before</option></term>
 
         <listitem>
-          <para>Show which units are started after or before
-          with <command>list-dependencies</command>, respectively.
+          <para>Show after (before) which units the specified unit is started
+          with <command>list-dependencies</command>.
           </para>
         </listitem>
       </varlistentry>
